themes/jquery.org/page-conduct.php

```diff
@@ -0,0 +1,50 @@
+<?php
+/**
+ * Template Name: Code of Conduct Sidebar Template
+ * Description: A Page Template that adds a sidebar to pages
+ */
+
+get_header(); ?>
+
+		<div id="primary" class="twelve columns">
+			<div id="content" role="main">
+
+				<?php the_post(); ?>
+				<?php if ( !count( get_post_meta( $post->ID, "hide_title" ) ) ) : ?>
```

Comment:
I think I copied this from a different template - I also haven't done PHP in forever but I think it could/should be single quotes -- not sure if we had a PHP style guide :)